### ISSUE TITLE
Localization required?

### ISSUE THREAD

**Issue by Digawen** [markjsosnowski/BlueMageHelper#1]
Is it possible that this plugin only works with the English client?
I play with the German client and get only "no data for spell #r. "

**Comment by markjsosnowski** [comment]
Yes, it only works in English right now. I have some plans for localization or some multiple language solution, probably will be next time I update it, whenever that is. Can you provide a screenshot of your whole blue mage spellbook? It would help.

**Comment by Digawen** [comment]
![1](https://user-images.githubusercontent.com/4532046/143320473-b780b0d2-2de1-4cab-9769-dc661c1a6367.png)


like this? from all pages?

**Comment by markjsosnowski** [comment]
What you posted is all I needed. Thank you.
